client/reader/discover/discover-stream.js

```diff
@@ -127,16 +133,14 @@ const DiscoverStream = ( props ) => {
 	};
 
 	return (
-		<>
-			<Stream { ...streamProps }>
-				{ DiscoverHeader() }
-				<DiscoverNavigation
-					width={ props.width }
-					selectedTab={ selectedTab }
-					recommendedTags={ interestTags }
-				/>
-			</Stream>
-		</>
+		<Stream { ...streamProps }>
+			<DiscoverHeader selectedTab={ selectedTab } width={ props.width } />
+			<DiscoverNavigation
```

Comment:
I assume it's expected that on SSR this won't load because we're doing this for the search engines?

![Screenshot 2024-09-27 at 18 35 23](https://github.com/user-attachments/assets/d5464c52-53aa-42e5-82d2-ecea9c4a3b08)

Reply:
Yes, this is how we intended it. The main goal of enabling SSR is indeed to improve SEO, by rendering the localized meta description and the `Descubre` and `Explora [...]` texts. Other than that, we wanted to keep the SSR light and not render the feed itself.

We thought showing this `PostPlaceholder` would be a small, user-friendly touch to indicate that content will be loaded there.

Do you think this is a good UX or would you change it? Thanks!

Reply:
Sounds good for this pass 👍 In combination with the "please enable JS" popup, it should be enough.